src/pages/patterns/empty-state-pattern/index.mdx

```diff
@@ -0,0 +1,481 @@
+<AnchorLink>Related patterns and components</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Empty states are a simple yet extremely powerful way to keep a user informed, supported, and on a productive path. They provide opportunities to communicate what the user would see if they had data, while providing constructive guidance about next steps.
+
+With just enough contextual guidance, empty states ensure a smoothness of experience, especially when things aren't working as expected.
+
+Most people are familiar with the basic empty state page that explains what data would normally appear on a page. However, it's not always a one size fits all.
+
+This pattern explores the following approaches:
+
+- Basic empty states for first use, user action confirmation, and error management
+- In-depth supplements and alternatives for first use empty states, including in-line documentation, onboarding, and starter content
+
+## Anatomy of an empty state
+
+<Row>
+<Column colLg={8}>
+
+![Basic empty state with primary action button](/images/1.png)
+
+<Caption>Basic empty state with primary action button</Caption>
+
+</Column>
+</Row>
+
+**1. Image:** (optional): A non-interactive image that relates to the situation (optional).
+
+**2. Title:** A short and concise explanation. Where possible, write this as a positive statement. In this example, "Start by adding data assets" feels more positive than "You don't have any data assets." Alternatively, you could say "You don't have any data assets yet".
+
+**3. Body:** Explain clearly the next action to populate the space. You may also explain why the space is empty and include the benefit of taking this step.
+
+There are three options for explaining the primary action:
+
+- direct the user to a primary action button positioned underneath the copy,
+- include a primary action link in the copy, or
+- direct the user to the UI element—see the example below. This has the benefit of teaching the user where elements are and how they will perform tasks in the future.
+
+**4. Primary action—button or link in copy (optional):** The primary call to action referenced in the body copy above.
+
+**5. Secondary call to action (optional):** If there is a secondary action, such as referencing documentation for further reading, include it as a link below the copy.
+
+<Row>
+<Column colLg={8}>
+
+![Basic empty state with instruction to click UI element](/images/2.png)
+
+<Caption>Basic empty state with instruction to click UI element</Caption>
+
+</Column>
+</Row>
+
+## Designing with empty states
+
+Empty states are often treated as an afterthought. When designed thoughtfully, they become an essential part of a smooth user experience, providing just enough information, in context, to allow users to continue working in a productive way.
+
+During the design process, ask yourself these questions:
+
+- What do the pages, tiles, datatables, and side panels look like without content?
```

Comment:
data tables

Reply:
> data tables

done

Reply:
Instead of "What do the pages", consider "What will the pages, …"? 
It more strongly conveys the kind of inevitability the broader messaging carries. There are going to be empty states at the beginning and possibly again and again in a product experience - we're advocating that they plan for it!